fastlane/Fastfile

```diff
@@ -0,0 +1,55 @@
+  # This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+default_platform(:android)
+
+platform :android do
+  appcenter_key = UI.input "Appcenter API Token: "
```

Comment:
Can we pull this from an env variable?

Reply:
Good catch, I'll have it seek an env var and fall back to prompting the user in the case it isn't set.